### PR TITLE
Fix hint and explanation flickering on drag and drop exercises

### LINKED ISSUE #3743: Quiz Hints/Explanations hover broken
### Describe the bug

When hovering a drag and drop quiz hint/explanation, the tooltip is shown and immediately disappears because it shoves itself in front of the mouse cursor.
This makes it almost impossible to read on large displays

### To Reproduce

Hover hint in drag and drop quiz

### Expected behavior

the tooltip should persist as long as the mouse is somewhere over the text

### Screenshots

https://user-images.githubusercontent.com/44805696/125434139-d8963fa9-1e38-4ed0-877c-ee64e7c752c1.mp4

### What browsers are you seeing the problem on?

Chrome

### Additional context

(Chromium)

### PR DESCRIPTION
**Pull request by joschahenningsen**
### Checklist
- [X] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [X] Client: I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [X] Client: I added multiple screenshots/screencasts of my UI changes

### Motivation and Context

fixes #3743 

### Steps for Testing

1. Create a drag and drop quiz with a hint and an explanation
2. preview quiz
3. verify that you are able to see the hints/explanations on hover.

### Test Coverage
unchanged

### Screencast

https://user-images.githubusercontent.com/44805696/125445904-1895db4c-23d4-491c-9eb5-7dba184ed123.mp4

### CODE REVIEW

File: src/main/webapp/app/exercises/quiz/shared/questions/drag-and-drop-question/drag-and-drop-question.component.html [hunk 1]
```diff
@@ -28,20 +28,21 @@ <h4 class="question-title-display">
...
                 <span jhiTranslate="artemisApp.quizQuestion.hint"></span> </span
-            ><br />
-            <ng-template #renderedExplanation>
-                <div [innerHTML]="renderedQuestion.explanation"></div>
+            >
+            <ng-template #renderedHint>
+                <div [innerHTML]="renderedQuestion.hint"></div>
             </ng-template>
-            <span class="label label-primary" [ngbPopover]="renderedExplanation" placement="right" triggers="mouseenter:mouseleave" *ngIf="question.explanation && showResult">
+            <br />
```
Comment (SimonEntholzer) [inline src/main/webapp/app/exercises/quiz/shared/questions/drag-and-drop-question/drag-and-drop-question.component.html:38]:
```suggestion
            <br/>
```
I guess here you missed a whitespace

**Review by SimonEntholzer — CHANGES_REQUESTED** [review]
some minor sytle issue with spaces, but otherwise looks good

**Review by SimonEntholzer — APPROVED** [review]
looks good to me.

**Review by ArtemTsurikov — CHANGES_REQUESTED** [review]
I still had the flickering issue on TS1. Maybe it was due to the background picture dimension size (1200 × 630). 

https://user-images.githubusercontent.com/64026687/125504829-c1d025c3-e587-4981-a5b9-f5d2cea51d53.mov

**Comment by joschahenningsen** [comment]
https://github.com/ls1intum/Artemis/pull/3746#pullrequestreview-705510883 
Thanks! I can't reproduce this with any image. Which browser did you use?

**Review by willergeorg — APPROVED** [review]
Tested on TS1. I wasn't able to reproduce the issue of @ArtemTsurikov, it worked good for me.

**Pull request merged**